microsoft/testsuites/cvm/cvm_boot.py

```diff
@@ -107,16 +111,25 @@ def verify_boot_success_after_component_upgrade(
         boot_components = ["shim", "systemd-boot", "kernel-uki"]
         boot_components_versions: Dict[str, str] = dict()
         for pkg in boot_components:
-            pkg_version = posix_os.get_package_information(pkg, use_cached=False)
+            pkg_version = os.get_package_information(pkg, use_cached=False)
             boot_components_versions[pkg] = pkg_version.version_str
 
+        repo_url = variables.get("rpm_repository_url")
```

Comment:
How about just use `rpm_repository`, remove the `rpm_source_tarball`? It looks like one repo var should be enough for remote or local.

Reply:
that's fair. Updated